composer.json

```diff
@@ -29,12 +32,19 @@
     },
     "scripts": {
         "tests": "vendor/bin/phpunit",
-        "build": [
-            "@tests"
+        "phpstan": "phpstan analyse --no-progress --level=max -c phpstan.neon src tests",
+        "phpcs": "phpcs --extensions=php src tests",
+        "phpcbf": "phpcbf --extensions=php src tests",
+        "phplint": "parallel-lint -j 10 --exclude vendor .",
+        "codestyle": [
```

Comment:
šlo by se držet toho jak je to v generatoru? https://github.com/keboola/component-generator/blob/master/templates/php-component/composer.json#L34

Reply:
Jestli myslíš, že to má nějaké výhody, tak propadnout skrz #best-practices a PR do component generátoru

Reply:
JJ, napadlo ma to. Mne pride dobre, mat moznost pustit tie "lintery" samostatne. Teda spustit vsetky ale bez testov alebo naopak pustit len testy. Spravim to ako pises ;)

Reply:
Já ve výsledku moc nevím. phpcs i phplint máš v PhpStormu. Takže zbývá jen phpstan, který bys třeba chtěl pustit extra, no...

Reply:
aha, ja to z PhpStormu nepustam, asi je to zbytocne sa tym zaoberat. Necham to zatial podla generatoru, keby ma to velmi sralo, tak vznesiem podnet v #best-practices ;)

Reply:
No, phpcs můžeš mít rovnou integrovaný v PhpStormu, že ti realtime podtrhává chyby. Nastavení na 2 minuty, jestli chceš, můžu ukázat - stačí call ;)